Dockerfile

```
@@ -44,10 +44,19 @@ RUN apt-get update && \
 #ENV HTTP_PROXY=${http_proxy} \
 #    HTTPS_PROXY=${https_proxy} \
 #    NO_PROXY=${no_proxy}
-#COPY MY_ORG_ROOT.crt /usr/local/share/ca-certificates
-#RUN cd /usr/local/share/ca-certificates && \
-#    wget http://pki.my-organization.local/MY%20ORG%20ROOT.crt && \
-#    update-ca-certificates
+
+# Handle custom CA certificate, if specified.
+RUN if test -n "${TRAM_CA_URL}" -a -n "${TRAM_CA_THUMBPRINT}" ; then \
+        echo "Installing certificate authority from ${TRAM_CA_URL}" && \
```

```suggestion
        set -o pipefail && echo "Installing certificate authority from ${TRAM_CA_URL}" && \
```

I think this is the right place given the piped command. This is the warning message from the linter:

`/github/workspace/Dockerfile:49 DL4006 warning: Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check`